metadata.txt

```diff
@@ -8,7 +8,7 @@
 
 [general]
 name=3di toolbox
-qgisMinimumVersion=2.14
```

Comment:
Some other parts of the plugin use function that are available from version 2.14 and up...